src/adc.rs

```diff
@@ -22,7 +22,7 @@
 //!     .assign(swm.pins.pio0_7.into_swm_pin(), &mut swm.handle);
 //!
 //! // Read a single value
-//! let adc_value = block! {adc.read(&mut adc_pin)}
+//! let adc_value = block!{adc.read(&mut adc_pin)}
```

Comment:
```suggestion
//! let adc_value = block!(adc.read(&mut adc_pin))
```

Never saw the `block!` macro being used with curly braces. Shouldn't make a difference, but this should reduce the chance of confusion for anyone new to Rust.